src/en/temp-release-notes.md

```diff
@@ -69,72 +71,70 @@ The "state-server" from Juju 1.x becomes a "controller" in 2.0.
-| juju environment set                  | juju set-model-config *** |
-| juju environment set-constraints      | juju set-model-constraints ** |
-| juju environment share                | juju share-model *** |
-| juju environment unset                | juju unset-model ** *** |
-| juju environment unshare              | juju unshare-model *** |
-| juju environment users                | juju list-shares |
-| juju user add                         | juju add-user |
-| juju user change-password             | juju change-user-password |
-| juju user credentials                 | juju get-user-credentials |
-| juju user disable                     | juju disable-user |
-| juju user enable                      | juju enable-user |
-| juju user info                        | juju show-user |
-| juju user list                        | juju list-users |
-| juju machine add                      | juju add-machine ** |
-| juju machine remove                   | juju remove-machine ** |
-| \<new in 2.0\>                        | juju list-machines |
-| \<new in 2.0\>                        | juju show-machines |
-| juju authorised-keys add              | juju add-ssh-key |
-| juju authorised-keys list             | juju list-ssh-keys |
-| juju authorised-keys delete           | juju remove-ssh-key |
-| juju authorised-keys import           | juju import-ssh-key |
-| juju get                              | juju get-config |
-| juju set                              | juju set-config |
-| juju get-constraints                  | juju get-model-constraints |
-| juju set-constraints                  | juju set-model-constraints |
-| juju get-constraints <application>    | juju get-constraints |
-| juju set-constraints <application>    | juju set-constraints |
-| juju backups create                   | juju create-backup *** |
-| juju backups restore                  | juju restore-backup *** |
-| juju action do                        | juju run-action *** |
-| juju action defined                   | juju list-actions *** |
-| juju action fetch                     | juju show-action-output *** |
-| juju action status                    | juju show-action-status *** |
-| juju storage list                     | juju list-storage *** |
-| juju storage show                     | juju show-storage *** |
-| juju storage add                      | juju add-storage *** |
-| juju space create                     | juju add-space *** |
-| juju space list                       | juju list-spaces *** |
-| juju subnet add                       | juju add-subnet *** |
-| juju ensure-availability              | juju enable-ha *** |
-
-\* the behaviour of destroy-environment/destroy-model has changed, see
-https://jujucharms.com/docs/devel/controllers
-\*\* these commands existed but are now the recommended approach
-\*\*\* this is an alias, but will be the primary command going forward
+    1.25 command                          2.0-beta command
```

Comment:
Curious why we are reformatting this from an easy-to-read table to a code block that is harder to read...